internal/formats/spdx22json/model/annotation.go

```diff
@@ -1,7 +1,5 @@
 package model
 
-import "time"
```

Comment:
Why are there changes to this file?

Reply:
See this comment here: https://github.com/anchore/syft/pull/941#discussion_r845580950